pandas/tests/indexes/multi/conftest.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+import pytest
+from pandas import Index, MultiIndex
+
+
+@pytest.fixture
+def _index():
```

Comment:
you don't need to prefix with '_'

Reply:
The non-fixture variable "index" is used in many of these tests.  Several of these also use the fixture "index" which I dubbed _index in an attempt to distinguish them.  

I suggest using something like m_index or index_fixture to provide some separation. 

Thoughts?